setup/docker-compose.yml

```diff
@@ -10,7 +10,7 @@ services:
      - ../data/ipfs/data:/data/ipfs
      - ../data/ipfs/staging:/export
   testrpc:
-    image: "harshjv/testrpc:latest"
+    image: "testprc_persist:latest"
```

Comment:
Where is this? I cannot seem to find an image with this name... or `testprc` or `testrpc_persist` for that matter